.gitignore

```diff
@@ -0,0 +1,29 @@
+.sass-cache
```

Comment:
Add GH recommendations for node

https://github.com/github/gitignore/blob/master/Node.gitignore